libs/execution_context/ebpf_native.c

```diff
@@ -1642,6 +1756,17 @@ ebpf_native_load_programs(
         goto Done;
     }
 
+    // Set initial map values.
+    result = _ebpf_native_set_initial_map_values(module);
```

Comment:
Do we know what is the expected behavior for "reused" maps? For "reused" maps, i.e. maps which are already present (and pinned), does loading of program expect the map entries to be overwritten with the static values?

Reply:
Given that the expected behavior is that it re-uses an existing map (doesn't create a new one), then this should remain the same irrespective of how the map is created? The function _ebpf_native_create_maps already takes care of this AFAIK and setting the initial value is downstream so it simply populates it with the map handles this module is using.

Reply:
After chatting I understand the concern. Loader should not overwrite values in existing maps.